### ISSUE TITLE
Ravie article is hard to parse due to missing line breaks. 

### ISSUE THREAD

**Issue by FontBoss** [MicrosoftDocs/typography-issues#841]
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 69a6fce6-b5d7-c127-210c-d4f2c1f10ca3
* Version Independent ID: 6d687f0d-c3e9-b5f0-b978-c61550c5d711
* Content: [Ravie font family - Typography](https://docs.microsoft.com/en-us/typography/font-list/ravie)
* Content Source: [typographydocs/font-list/ravie.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/font-list/ravie.md)
* Product: **typography**
* GitHub Login: @alib-ms
* Microsoft Alias: **alib**

**Comment by alib-ms** [comment]
Can you please check again as I have updated the description for Ravie. I am not seeing any issues with the page on my end. Thanks
